coldFilmSearchParser.js

```diff
@@ -0,0 +1,72 @@
+const htmlParser = require('htmlparser2');
+
+function coldFilmSearchParser(parseString){
+    let enableParseLink = false;
+    let enableParseDate = false;
+    let enableParseNextPage = false;
+
+    let nextPageLink = '';
+    let serialTitle = '';
+    let series = [];
+    let serial = {};
+
+    let parser = new htmlParser.Parser({
+        onopentag: (name, attribs) => {
```

Comment:
make separate functions `function onOpenTagCallback(name, attrs)` and pass it here
same for other callbacks